definitions/divorce/json/CaseEventToFields/CaseEventToFields-nonprod.json

```diff
@@ -0,0 +1,326 @@
+[
+  {
+    "LiveFrom": "20/09/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "EnterAOSOffline_CoRespondent",
+    "CaseFieldID": "D8PetitionerFirstName",
+    "PageFieldDisplayOrder": 99,
+    "DisplayContext": "READONLY",
+    "PageID": "aosdataentrycoresp1",
+    "PageLabel": "Enter scanned form case data",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "ReceivedAosFromCoRespDate=\"nevershow\"",
```

Comment:
I've copied the whole page, but it seems this is never shown.
I'm just taking the more cautious approach here.

Reply:
It seems to be showing actually:
<img width="723" alt="Screenshot 2020-05-21 at 09 22 00" src="https://user-images.githubusercontent.com/16866068/82883329-2cfe3c00-9f3a-11ea-9b12-294cb948f05a.png">

Reply:
Good spot. However, this seems to be the content of the "labelAOSOfflineAdulteryCoRespDetails" field.